src/main/java/eventprocessing/models/SensorList.java

```diff
@@ -1,16 +1,16 @@
-package eventprocessing.models;
-
-import java.util.List;
-
-public class SensorList {
-
-    private List<Sensor> sensors;
-
-    public SensorList(List<Sensor> sensors) {
-        this.sensors = sensors;
-    }
-
-    public List<Sensor> getSensors() {
-        return sensors;
-    }
-}
+package eventprocessing.models;
+
+import java.util.List;
+
+public class SensorList {
```

Comment:
This class doesn't do anything except wrap `List<Sensor>`. Sometimes wrapping a type is useful because you can give it a more domain-specific name, but `SensorList` is pretty much the same thing as `List<Sensor>` so perhaps it would be simpler to remove it?

Reply:
Having said that, there's a `isWorkingSensor` method below which _would_ make sense to move onto `SensorList`! So maybe you shouldn't delete it - maybe it's worth keeping around but pushing some functionality onto it?